listenbrainz/labs_api/labs/tests/test_year_from_artist_credit_recording.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import patch
+
+import flask_testing
+from datasethoster.main import create_app
+from listenbrainz.labs_api.labs.api.year_from_artist_credit_recording import YearFromArtistCreditRecordingQuery
+
+
+json_request = [
+    {
+        "[artist_credit_name]": "Portishead",
+        "[recording_name]": "Strangers"
+    },
+    {
+        "[artist_credit_name]": "Rick Astley",
+        "[recording_name]": "Never let you down"
+    },
+    {
+        "[artist_credit_name]": "morcheeba",
+        "[recording_name]": "trigger hippie"
+    }
+]
+
+json_response = [
+    {
+        "artist_credit_name": "Portishead",
+        "recording_name": "Strangers",
+        "year": 1994
+    },
+    {
+        "artist_credit_name": "Rick Astley",
+        "recording_name": "Never let you down",
+        "year": 1987
+    }
+]
+
+
+class MainTestCase(flask_testing.TestCase):
+
+    def create_app(self):
+        app = create_app()
+        app.config['MB_DATABASE_URI'] = 'yermom'
+        return app
+
+    def setUp(self):
```

Comment:
I think you can get rid of the `setUp` and `tearDown` methods because they only call the super class methods.